### PR TITLE
Show events and meetings on front page website using posts

### PR DESCRIPTION
**Pull request by dcwalk**
Beginning with just the data from our previous meetings, what remains:
- [x] feedback on YAML frontmatter 
- [x] add in method to show on front page

Will address #9 , having .ics file generated and creating an event page in a separate PR.

### CODE REVIEW

**Comment by dcwalk** [comment]
Looking at bringing in a CSS grid for layout, these two seem good candidates:
- [Skeleton](http://getskeleton.com/)
- [Simple Grid](https://thisisdallas.github.io/Simple-Grid/) 

Examples of how others have calendared:
- [jekyll-bootstrap-calendar](https://github.com/NLKNguyen/jekyll-bootstrap-calendar/blob/master/example/calendar.markdown)
- [seattle hacks generated json into .ics](https://www.twilio.com/blog/2013/04/building-the-noiseless-hacker-calendar-seattle-hacks.html) and on [github](https://github.com/crtr0/seattlehacks/blob/gh-pages/calendar.html), [example of YAML](https://raw.githubusercontent.com/crtr0/seattlehacks/gh-pages/_posts/2013-04-03-linux-fest-nw.md) _not sure if working?_
- [jekyll-google-calendar](https://gist.github.com/iani/8906741)
- [jekyll-calendar](https://github.com/budparr/jekyll-calendar)

File: _posts/2016-05-20-prototype-session.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+title: Outreach Work Session
+text: Join us at our work-session for Toronto Mesh outreach.
+location:
+locationMap: http://osm.org/go/ZX6BNoGot-?m=
+date: 2015-05-20
+startTime: '18:30'
+endTime: '21:30'
+---
+
+Join us at our work-session for the e. We are meeting at 6:30 pm at dcwalk's place, closest intersection is College and Ossington. OpenStreetMap  [directions](http://osm.org/go/ZX6BNoGot-?m=).
```
Comment (benhylau) [inline _posts/2016-05-20-prototype-session.md]: First line got broken?

Comment (dcwalk) [inline _posts/2016-05-20-prototype-session.md]: fixed, good catch!

**Comment by dcwalk** [comment]
Just a snapshot of this... would like to discuss before going too far in one direction... right now it will show all upcoming events on the front page, if there aren't any a little blurb will show.

screen shot 2016-05-24 at 1 35 58 pm

**Comment by garrying** [comment]
@dcwalk When you have moment, can you take a look at the changes I've made? There's an issue with showing more than 2 events on the page (bad wrapping) that I'm aware of. Do you think it's a case we should accommodate for?

**Comment by dcwalk** [comment]
@garrying willl check out! I have a couple thoughts but only skimmed...
- the .ics might not be valid (checked [here](http://severinghaus.org/projects/icv/))
- that button I added is confusing anyway, we should probably split ICS off to a separate PR
- we could use the post blogroll to show previous events and have THAT be linked from a button, that could also handle the problem with wrapping (have that page have all upcoming events at the top, JUST show the first two on the main page
- Do we need to address IA... e.g., having a global page menu at the top with events?

I'll have some time on Sunday to do a better top of review, and can push come changes. In the meantime thank you thank you thank you for refactor and sass goodness 💃

**Pull request merged**